### DISCUSSION TITLE
Checkbox field results loop in Templates

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#3363] [Feature Requests]
### Suggest an idea for ExpressionEngine

When outputting results from a Checkbox field, in an exp:channel:entries loop, I'd like more control over what's displayed.
I'd like to be able to achieve this:  
```
{school-type}
	{if count == 1}<ul class="schooltype">{/if}
		<li>item:value</li>
	{if count == total_results}</ul>{/if}
{/school-type}
```
...but it seems the total_results value is taking the total_results from the Entries, rather than the 'school-type' field.
As a workaround, I'm having to do this:
```
{school-type}
	{if count == 1}<ul class="schooltype">{/if}
		<li>item:value</li>
{/school-type}
{if school-type}</ul>{/if}
```

**Comment by intoeetive** [comment]
@ic360-com I can confirm that only the following variables are being set inside Checkboxes tag pair:
`item'
`item:label`
`item:value`
`count`

Besides `total_results` (which I think we will need to name differently to avoid breaking existing template code for people), which ones do you think would be useful?

**Comment by ic360-com** [comment]
Thanks for confirming.
I can't think of any further variables that I'd need, right now.

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/3800
